prac_05/emails.py

```diff
@@ -0,0 +1,38 @@
+"""
+CP1404/CP5632 Practical 05
+Emails
+Estimate: 25 minutes
+Actual:  minutes
+"""
+
+
+def main():
+    """Main program to store users emails and names in a dictionary"""
+    emails_to_names = {}
```

Comment:
maybe just me, but email_to_names sounds like a function not a library

Reply:
_"A reasonable way to name dictionaries (aka maps) is the key_to_value pattern, e.g., name_to_phone or level_to_reward (using something like phones or levels here would imply they are lists/sets/tuples/etc. not dictionaries)."_

I'm just following the [style guide](https://github.com/CP1404/Starter/wiki/Style-Guide) for this.